[CS2113T-T09-3] All-in-OneNUS

rageqqq opened the pull request.

fangxinjia0203 committed:
v1.1

fangxinjia0203 committed:
Merge pull request #28 from fangxinjia0203/ModulePlanner

v1.1

fangxinjia0203 committed:
coding standard

rageqqq committed:
v1.01

Fixed Java CI requirements
Added printWelcomeMessage and printAvailableCommands methods

fangxinjia0203 committed:
Merge pull request #30 from fangxinjia0203/ModulePlanner

coding standard

rageqqq committed:
Merge pull request #31 from rageqqq/v1.01

v1.01

fangxinjia0203 committed:
coding standard 2nd

fangxinjia0203 committed:
Merge pull request #32 from fangxinjia0203/ModulePlanner

coding standard 2nd

rageqqq committed:
coding standard

changed assertion import

soepaingzaw committed:
JUnit Coding standard

fangxinjia0203 committed:
coding standard 3rd

rageqqq committed:
Merge pull request #33 from rageqqq/codingstandard

coding standard

soepaingzaw committed:
Merge pull request #34 from soepaingzaw/CapCalculator-Branch

JUnit Coding standard

fangxinjia0203 committed:
Merge pull request #35 from fangxinjia0203/ModulePlanner

coding standard 3rd

douglaslewpc committed:
v1.2

douglaslewpc committed:
Merge pull request #36 from douglaslewpc/v1.2

v1.2

douglaslewpc committed:
v1.2.1

fangxinjia0203 committed:
4th

douglaslewpc committed:
Merge pull request #37 from douglaslewpc/v1.2

v1.2.1

fangxinjia0203 committed:
Merge pull request #38 from fangxinjia0203/ModulePlanner

4th

douglaslewpc committed:
v1.2.2

douglaslewpc committed:
Merge pull request #39 from douglaslewpc/v1.2

v1.2.2

rageqqq committed:
Update runtest.sh

rageqqq committed:
Merge pull request #40 from rageqqq/permission

Update runtest.sh

fangxinjia0203 committed:
add storage

fangxinjia0203 committed:
Merge pull request #41 from fangxinjia0203/ModulePlanner

add storage

fangxinjia0203 committed:
better ui

fangxinjia0203 committed:
Merge pull request #42 from fangxinjia0203/ModulePlanner

better ui

douglaslewpc committed:
v1.0

douglaslewpc committed:
Merge pull request #43 from douglaslewpc/master

v1.0

douglaslewpc committed:
v1.0

douglaslewpc committed:
Merge pull request #44 from douglaslewpc/heads/v1.0

v1.0

rageqqq committed:
v1.02

Added parser and exceptions

rageqqq committed:
Merge pull request #45 from rageqqq/v1.02

v1.02

douglaslewpc committed:
v1.0

soepaingzaw committed:
partial parser

soepaingzaw committed:
Merge pull request #46 from soepaingzaw/CapCalculator-Branch

partial parser

douglaslewpc committed:
Merge pull request #47 from douglaslewpc/heads/v1.0

v1.0

douglaslewpc committed:
v1.01

douglaslewpc committed:
Merge pull request #48 from douglaslewpc/v1.01

v1.01

rageqqq committed:
v1.0

added food recommendation to overallparser

rageqqq committed:
Merge pull request #49 from rageqqq/v1.0

v1.0

rageqqq committed:
v1.0

removed assertion

rageqqq committed:
Merge pull request #50 from rageqqq/v1.0

v1.0

rageqqq committed:
added exit command for overallparser

rageqqq committed:
Merge pull request #51 from rageqqq/v1.0

added exit command for overallparser

fangxinjia0203 committed:
not completed

rageqqq committed:
v1.0

added messages to acknowledge commands are received

rageqqq committed:
Merge pull request #52 from rageqqq/v1.0

v1.0

fangxinjia0203 committed:
v1.0 xinjia

fangxinjia0203 committed:
Merge branch 'master' into ModulePlanner

fangxinjia0203 committed:
Merge pull request #53 from fangxinjia0203/ModulePlanner

v1.0 xinjia

rageqqq committed:
v1.0

added storage text

rageqqq committed:
Merge pull request #54 from rageqqq/v1.0-chester

v1.0

douglaslewpc committed:
v1.0

douglaslewpc committed:
Merge pull request #55 from douglaslewpc/master

v1.0

soepaingzaw committed:
multiple refactoring of classes and creation of main file

soepaingzaw committed:
Merge pull request #56 from soepaingzaw/CapCalculatorV1.0-Branch

multiple refactoring of classes and creation of main file

fangxinjia0203 committed:
userguide

soepaingzaw committed:
assertion successful

soepaingzaw committed:
clear checkstyle of cap calculator

soepaingzaw committed:
Merge pull request #58 from soepaingzaw/CapCalculatorV1.0-Branch

douglaslewpc committed:
v1.0

douglaslewpc committed:
Merge pull request #59 from douglaslewpc/heads/v1.0

v1.0

douglaslewpc committed:
v1.0

douglaslewpc committed:
Merge pull request #60 from douglaslewpc/heads/v1.0

v1.0

fangxinjia0203 committed:
Merge branch 'master' of https://github.com/AY2021S2-CS2113T-T09-3/tp into ModulePlanner

# Conflicts:
#	src/main/java/seedu/allinonenus/OverallParser.java

fangxinjia0203 committed:
Merge branch 'master' of https://github.com/AY2021S2-CS2113T-T09-3/tp into ModulePlanner

fangxinjia0203 committed:
fix bugs for modplanner

fangxinjia0203 committed:
Merge pull request #61 from fangxinjia0203/ModulePlanner

Module planner

fangxinjia0203 committed:
hyperlink

fangxinjia0203 committed:
hyperlink 2

soepaingzaw committed:
added storage files

soepaingzaw committed:
Merge branch 'master' of https://github.com/AY2021S2-CS2113T-T09-3/tp into CapCalculatorV1.0-Branch

# Conflicts:
#	ModulePlanner.txt

soepaingzaw committed:
enable assertions

soepaingzaw committed:
Merge pull request #62 from soepaingzaw/CapCalculatorV1.0-Branch

Cap calculator v1.0 branch

soepaingzaw committed:
implement sem information

soepaingzaw committed:
Implement show all function

soepaingzaw committed:
Merge pull request #63 from soepaingzaw/CapCalculatorV1.0-Branch

Cap calculator v1.0 branch

fangxinjia0203 committed:
hyperlink 3

fangxinjia0203 committed:
hyperlink 4

fangxinjia0203 committed:
Merge pull request #64 from fangxinjia0203/ModulePlanner

hyperlink

rageqqq committed:
v1.0

Updated UG

rageqqq committed:
Merge pull request #65 from rageqqq/v1.0-UG

v1.0

rageqqq committed:
v1.0

updated UG again

rageqqq committed:
Merge pull request #66 from rageqqq/v1.0-UG

v1.0

douglaslewpc committed:
UG

douglaslewpc committed:
Merge pull request #67 from douglaslewpc/UG

UG

rageqqq committed:
v1.0

updated UG AGAIN

rageqqq committed:
Merge pull request #68 from rageqqq/v1.0-UG

v1.0

douglaslewpc committed:
UG

douglaslewpc committed:
Merge pull request #69 from douglaslewpc/master

UG

douglaslewpc committed:
UG

douglaslewpc committed:
Merge pull request #70 from douglaslewpc/UG

UG

douglaslewpc committed:
UG

douglaslewpc committed:
Merge pull request #71 from douglaslewpc/UG

UG

rageqqq committed:
v1.0-ug

rageqqq committed:
Merge pull request #72 from rageqqq/v1.0-ug

v1.0-ug

soepaingzaw committed:
DG add image and draft UML diagram

soepaingzaw committed:
Merge pull request #73 from soepaingzaw/CapCalculatorV1.0-Branch

DG add content and draft UML diagram

soepaingzaw committed:
added seperation lines between different features

rageqqq committed:
dg

rageqqq committed:
Merge pull request #75 from rageqqq/DG

dg

soepaingzaw committed:
Merge pull request #76 from soepaingzaw/CapCalculatorV1.0-Branch

added separation lines between different features

soepaingzaw committed:
Update names in team

soepaingzaw committed:
Merge pull request #78 from soepaingzaw/CapCalculatorV1.0-Branch

Update names in team

douglaslewpc committed:
Update DeveloperGuide.md

Updated DG with UML Diagram for TeamPlanner Class

douglaslewpc committed:
Update DeveloperGuide.md

douglaslewpc committed:
Update DeveloperGuide.md

douglaslewpc committed:
Update AboutUs.md

Added picture and GitHub Link

douglaslewpc committed:
Update AboutUs.md

douglaslewpc committed:
Update AboutUs.md

douglaslewpc committed:
Update AboutUs.md

Updated Picture

douglaslewpc committed:
Update AboutUs.md

fangxinjia0203 committed:
uml

fangxinjia0203 committed:
Update DeveloperGuide.md

fangxinjia0203 committed:
Merge branch 'master' into ModulePlanner

rageqqq committed:
Update AboutUs.md

soepaingzaw committed:
update About us

soepaingzaw committed:
Merge pull request #80 from soepaingzaw/CapCalculatorV1.0-Branch

update About us

soepaingzaw committed:
update about us

soepaingzaw committed:
Merge pull request #81 from soepaingzaw/CapCalculatorV1.0-Branch

update about us

soepaingzaw committed:
resize picture

soepaingzaw committed:
Merge pull request #82 from soepaingzaw/CapCalculatorV1.0-Branch

resize picture

soepaingzaw committed:
store current sem of student

douglaslewpc committed:
Added feature to add/delete tasks and modifited various classes

douglaslewpc committed:
Merge pull request #83 from douglaslewpc/master

Added feature to add/delete tasks and modifited various classes

soepaingzaw committed:
clean up format for Cap Calculator

soepaingzaw committed:
Merge branch 'master' of https://github.com/AY2021S2-CS2113T-T09-3/tp into CapCalculatorV1.0-Branch

soepaingzaw committed:
added update feature

soepaingzaw committed:
finished Cap Calculator features

soepaingzaw committed:
Merge pull request #84 from soepaingzaw/CapCalculatorV1.0-Branch

Cap calculator v1.0 branch

rageqqq committed:
v2.0

Added locations, which are seperated by sub classes
Added find command, can find stores by keyword
Added halal command, can change whether the store is halal or not

rageqqq committed:
no message

rageqqq committed:
Merge pull request #85 from rageqqq/v2.0

V2.0

soepaingzaw committed:
improve code architecture

soepaingzaw committed:
Merge branch 'master' of https://github.com/AY2021S2-CS2113T-T09-3/tp into CapCalculatorV1.0-Branch

soepaingzaw committed:
update architecture

soepaingzaw committed:
V2.0

soepaingzaw committed:
Merge pull request #86 from soepaingzaw/CapCalculatorV1.0-Branch

Cap calculator v1.0 branch

rageqqq committed:
Improve code for Java CI

rageqqq committed:
Merge pull request #87 from rageqqq/v2.0

Improve code for Java CI

rageqqq committed:
Improve code for Java CI

rageqqq committed:
Merge pull request #88 from rageqqq/v2.0

Improve code for Java CI

rageqqq committed:
Fixed no main manifest attribute by moving manifest.mnf to java/main/resources

rageqqq committed:
Merge pull request #89 from rageqqq/v2.0

Fixed no main manifest attribute by moving manifest.mnf to java/main/…

rageqqq committed:
Update DeveloperGuide.md

soepaingzaw committed:
update checkstyle

soepaingzaw committed:
DG update

soepaingzaw committed:
Adding diagrams for Cap Calculator Implementation

soepaingzaw committed:
Merge pull request #90 from soepaingzaw/CapCalculatorV1.0-Branch

Cap calculator v1.0 branch

soepaingzaw committed:
minor update

soepaingzaw committed:
Merge pull request #91 from soepaingzaw/CapCalculatorV1.0-Branch

minor update

fangxinjia0203 committed:
Merge branch 'master' of https://github.com/AY2021S2-CS2113T-T09-3/tp into ModulePlanner

# Conflicts:
#	docs/UserGuide.md

fangxinjia0203 committed:
Merge branch 'ModulePlanner' of https://github.com/fangxinjia0203/tp into ModulePlanner

# Conflicts:
#	docs/UserGuide.md

douglaslewpc committed:
v2.0

douglaslewpc committed:
Merge pull request #92 from douglaslewpc/master

v2.0

fangxinjia0203 committed:
fix bugs and add delete student for moduleplanner

fangxinjia0203 committed:
Merge pull request #79 from fangxinjia0203/ModulePlanner

fixed bugs and add delete for moduleplanner

fangxinjia0203 committed:
fix bugs

fangxinjia0203 committed:
Merge pull request #93 from fangxinjia0203/ModulePlanner

fix bugs

Chiamjiaen reviewed (commented):
![image](https://user-images.githubusercontent.com/64964101/113426759-8494f000-9406-11eb-8c3d-fac18e3fd23d.png)
This particular image has very small sections and is extremely hard to see

Chiamjiaen reviewed (commented):
![image](https://user-images.githubusercontent.com/64964101/113426851-9d9da100-9406-11eb-94b6-be3d2fb5975b.png)
Image seems to be broken and there seems to be more than 1 line to demarked sections which seems to be inconsistent.

Chiamjiaen reviewed (commented):
![image](https://user-images.githubusercontent.com/64964101/113426958-c6be3180-9406-11eb-99ac-e8d0afb1b3f9.png)
This section seems to be incomplete

Chiamjiaen reviewed (commented):
![image](https://user-images.githubusercontent.com/64964101/113427034-e8b7b400-9406-11eb-8c61-1e3c7e75c1f6.png)
Better indentations can be used to make sections clearer

Chiamjiaen reviewed (commented):
The headers sizes seem inconsistent and more can be done to improve the clarity of certain sections.

rageqqq committed:
Added exception

rageqqq committed:
Merge pull request #94 from rageqqq/heads/v2.0

Added exception

soepaingzaw committed:
update UG and tweak code

soepaingzaw committed:
minor changes to UG

soepaingzaw committed:
Merge pull request #95 from soepaingzaw/CapCalculatorV1.0-Branch

minor updates

soepaingzaw committed:
changes for demo

douglaslewpc committed:
Updated UG with TeamPlanner features

douglaslewpc committed:
Merge pull request #165 from douglaslewpc/master

Updated UG with TeamPlanner features

douglaslewpc committed:
no message

douglaslewpc committed:
Merge pull request #166 from douglaslewpc/master

no message

soepaingzaw committed:
Increase readability of app

soepaingzaw committed:
Merge pull request #167 from soepaingzaw/CapCalculatorV1.0-Branch

update readability

andyrobert3 commented:
![image](https://user-images.githubusercontent.com/15076300/113504881-cbe9c080-956d-11eb-9bbf-91040969e864.png)

The diagram above is a good start for the draft DG, moving forward maybe adding granularity to the architecture can benefit others who are reading it to have a deeper understanding of each component

andyrobert3 commented:
![image](https://user-images.githubusercontent.com/15076300/113504980-7eba1e80-956e-11eb-9d94-855559df52ce.png)

To make it easier to understand, maybe it would be better to write a short description of the whole architecture on how it runs. Then possibly write a short description of each of the components on their functionalities.

andyrobert3 commented:
![image](https://user-images.githubusercontent.com/15076300/113505042-ff791a80-956e-11eb-8b59-c81dfae40a4a.png)

What do the multiple partitions indicate here? 
Class diagrams should contain only 3 partitions:
- Class Name
- Attributes
- Methods

Double checking the output of an online tool would help significantly

andyrobert3 commented:
![image](https://user-images.githubusercontent.com/15076300/113505124-79110880-956f-11eb-9e73-2b6c5d4b73f5.png)

What does <<create>> refer to?
If it is a constructor, then we can just stick to the standards in class on how to draw a constructor call

andyrobert3 commented:
![image](https://user-images.githubusercontent.com/15076300/113505167-becdd100-956f-11eb-85cc-22d2cd1577b9.png)

Try double checking on the dashed and straight lines, computer-generated drawings are not always the most reliable

andyrobert3 commented:
![image](https://user-images.githubusercontent.com/15076300/113505201-e6bd3480-956f-11eb-99fc-dcfaddae5b60.png)

You may want to add Sequence diagrams for the implementation part

andyrobert3 commented:
![image](https://user-images.githubusercontent.com/15076300/113505226-07858a00-9570-11eb-8c8e-8f736262f0fd.png)

Try to ensure its consistent with the other class diagrams where it has three partitions, instead of only one partition representing the class name

andyrobert3 commented:
![image](https://user-images.githubusercontent.com/15076300/113505263-3c91dc80-9570-11eb-9f6a-5e244632b9d5.png)

What does the quick overview entail here? Could you clarify more on that

andyrobert3 commented:
![image](https://user-images.githubusercontent.com/15076300/113505276-559a8d80-9570-11eb-9c21-b97ec58743f1.png)

Is this a Class Diagram? For implementation, I would suggest using Sequence diagrams to show how it works instead of showing how everything fits together

andyrobert3 commented:
![image](https://user-images.githubusercontent.com/15076300/113505293-69de8a80-9570-11eb-8024-652f46ff0bda.png)

Is this supposed to be the intended behaviour?

andyrobert3 commented:
![image](https://user-images.githubusercontent.com/15076300/113505308-824ea500-9570-11eb-9c37-7942d64d6d0d.png)

Comments same as above
1. Ensure the lines are broken/unbroken as expected
2. <<create>> can be replaced with the constructor call to make the diagrams more consistent
3. Implementation not done yet

andyrobert3 commented:
![image](https://user-images.githubusercontent.com/15076300/113505694-1b7ebb00-9573-11eb-8f4c-de17740e9695.png)

This part does not seem done yet

douglaslewpc committed:
v2.01

douglaslewpc committed:
Merge pull request #168 from douglaslewpc/master

v2.01

soepaingzaw committed:
update edit command

soepaingzaw committed:
Merge pull request #169 from soepaingzaw/CAPCalculatorMain-Branch

update edit command

soepaingzaw committed:
PPP draft

soepaingzaw committed:
Merge pull request #170 from soepaingzaw/CAPCalculatorMain-Branch

PPP draft

soepaingzaw committed:
empty line and incorrect sem handling

fangxinjia0203 committed:
minor bugs fixed

fangxinjia0203 committed:
Merge branch 'master' of https://github.com/AY2021S2-CS2113T-T09-3/tp into ModulePlanner

fangxinjia0203 committed:
ppp

fangxinjia0203 committed:
Merge pull request #171 from fangxinjia0203/ModulePlanner

Module planner

douglaslewpc committed:
Set theme jekyll-theme-cayman

soepaingzaw committed:
Merge branch 'master' of https://github.com/AY2021S2-CS2113T-T09-3/tp into CAPCalculatorMain-Branch

soepaingzaw committed:
PE prep

soepaingzaw committed:
Merge pull request #174 from soepaingzaw/CAPCalculatorMain-Branch

Cap calculator main branch

soepaingzaw committed:
Update main menu

soepaingzaw committed:
Merge pull request #175 from soepaingzaw/CAPCalculatorMain-Branch

Update main menu

soepaingzaw committed:
reformat links for better readability

soepaingzaw committed:
Merge pull request #176 from soepaingzaw/CAPCalculatorMain-Branch

reformat links for better readability

soepaingzaw committed:
minor tweak to UG

soepaingzaw committed:
Merge pull request #177 from soepaingzaw/CAPCalculatorMain-Branch

minor tweak to UG

soepaingzaw committed:
enable hyperlink and add Command summary

soepaingzaw committed:
Merge pull request #178 from soepaingzaw/CAPCalculatorMain-Branch

enable hyperlink and add Command summary

soepaingzaw committed:
update hyperlink

soepaingzaw committed:
Merge pull request #179 from soepaingzaw/CAPCalculatorMain-Branch

update hyperlink

soepaingzaw committed:
minor changes to UG

soepaingzaw committed:
added sample outputs to UG

soepaingzaw committed:
Merge pull request #180 from soepaingzaw/CAPCalculatorMain-Branch

added sample outputs to UG

soepaingzaw committed:
finalised CAP Calculator UG

soepaingzaw committed:
added test code

soepaingzaw committed:
add more Exceptions, logging and assertions

soepaingzaw committed:
Merge pull request #181 from soepaingzaw/CAPCalculatorMain-Branch

Cap calculator main branch

soepaingzaw committed:
update DG

soepaingzaw committed:
Merge pull request #182 from soepaingzaw/CAPCalculatorMain-Branch

update DG

soepaingzaw committed:
fix capitalisation problem

soepaingzaw committed:
Merge pull request #183 from soepaingzaw/CAPCalculatorMain-Branch

fix capitalisation problem

douglaslewpc committed:
Edited code, UG and ReadME documents

douglaslewpc committed:
Merge pull request #184 from douglaslewpc/branch-v2.1

Edited code, UG and ReadME documents

rageqqq committed:
add error message for unknown command in main food bot screen
show bye command to user to leave the food bot
fix delete and halal command to handle empty index, negative and 0 values
fix add command to handle empty descriptions (d/ followed by nothing)

rageqqq committed:
Merge pull request #185 from rageqqq/v2.1_CS

v2.1 bug fixes

douglaslewpc committed:
Edited UG

douglaslewpc committed:
Merge pull request #186 from douglaslewpc/branch-v2.1

Edited UG

douglaslewpc committed:
Updated Command Summary Table

douglaslewpc committed:
Merge pull request #187 from douglaslewpc/branch-v2.1

Updated Command Summary Table

douglaslewpc committed:
Updated UG

douglaslewpc committed:
Merge pull request #188 from douglaslewpc/branch-v2.1

Updated UG

rageqqq committed:
added to ug

rageqqq committed:
Merge pull request #189 from rageqqq/v2.1_csug

added to ug

rageqqq committed:
updated summary and index

rageqqq committed:
Merge pull request #190 from rageqqq/v2.1_csss

updated summary and index

soepaingzaw committed:
updates to Cap Calculator DG

soepaingzaw committed:
Merge pull request #191 from soepaingzaw/CAPCalculatorMain-Branch

updates to Cap Calculator DG

soepaingzaw committed:
code quality CAP Calculator

soepaingzaw committed:
Merge pull request #192 from soepaingzaw/CAPCalculatorMain-Branch

code quality CAP Calculator

rageqqq committed:
updated dg with design

soepaingzaw committed:
change about us PPP link

rageqqq committed:
Merge pull request #193 from rageqqq/v2.1_csss

updated dg with design

soepaingzaw committed:
Merge pull request #194 from soepaingzaw/CAPCalculatorMain-Branch

change about us PPP link

douglaslewpc committed:
Update DeveloperGuide.md

douglaslewpc committed:
Update DeveloperGuide.md

douglaslewpc committed:
Update DeveloperGuide.md

soepaingzaw committed:
soe paing's PPP

rageqqq committed:
updated implementation

soepaingzaw committed:
Merge pull request #196 from soepaingzaw/CAPCalculatorMain-Branch

soe paing's PPP

rageqqq committed:
Merge branch 'master' into v2.1csdg

rageqqq committed:
Merge pull request #195 from rageqqq/v2.1csdg

updated implementation

rageqqq committed:
updated ppp

rageqqq committed:
Merge pull request #197 from rageqqq/v2.1cssss

updated ppp

douglaslewpc committed:
PPP document

douglaslewpc committed:
Merge pull request #198 from douglaslewpc/branch-v2.1

PPP document

douglaslewpc committed:
Update douglaslewpc.md

douglaslewpc committed:
Update AboutUs.md

douglaslewpc committed:
Update AboutUs.md

douglaslewpc committed:
Update AboutUs.md

douglaslewpc committed:
Update AboutUs.md

douglaslewpc committed:
Update AboutUs.md

douglaslewpc committed:
Update AboutUs.md
